docs/websocket/feedback.md

```diff
@@ -63,14 +63,30 @@ the following json is sent as a progressive result:
 ```json
 {
   "sentence": 0,
-  "feedback": true
+  "feedback": true,
+  "words": [
+    {
+      "expected": "gif",
+      "recognized": "jif",
+      "correct": false
```

Comment:
This is ok, I'm just wondering if this covers all situation, I don't think so.
AFAIK, these are possibilities: COR: correct, DEL: deletion, INS: insersion, SUB: substitution

Reply:
Let's keep this PR open for now, until we've tested this a bit more.

Reply:
Correct: 
```
{
  "expected": "spam",
  "recognized": "spam",
  "correct": true
}
```

Deletion:
```
{
  "expected": "spam",
  "recognized": "",
  "correct": false
}
```

Insertion:
```
{
  "expected": "",
  "recognized": "spam",
  "correct": false
}
```

Substitution:
```
{
  "expected": "spam",
  "recognized": "eggs",
  "correct": false
}
```

As discussed in this mornings standup I'll add an extra field to explicitly state the error.